adapter-http/pom.xml

```diff
@@ -41,6 +41,11 @@
             <version>${revision}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.10.1</version>
```

Comment:
minor detail for after, this version should be left to mvn-default management section.